### PR TITLE
Add badges, contributing, and release guideline to readme

### PR DESCRIPTION
**Pull request by cducrest**
Taken from relay readme

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+Please include a description of what you would like to see implemented and an explanation of why you believe this would
+ be a good addition.
+
+## Open a pull request
+
+### Before starting
+- Follow the [readme](/README.rst) to setup you dev environment.
+- Ensure that you can run the tests and that they pass. The tests can be run from the root with
+`./pytest tests`
```
Comment (berndbohmeier) [inline CONTRIBUTING.md:31]: `pytest tests` or maybe even better `make tests` here

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  Then follow the instructions of the end2end repository to run the tests with that image.
+
+### To include
+
+- For a bug fix:
+  - Add a test that makes the bug explicit, and make sure that the test fails
+  - Fix the bug, so that the test passes
+- For a new feature:
+  - Add a test, that demonstrate the new feature. Consider also adding the new functionality
+  together with an end2end test to the
```
Comment (berndbohmeier) [inline CONTRIBUTING.md:45]: I think here we should also mention the relay.

File: README.rst [hunk 3]
```diff
@@ -91,15 +98,15 @@ Finally, to install all needed dependencies and compiling the contracts use the
...
-- The code is formatted with black.
-- Commit messages are written following these
-  `guidelines
-  <https://chris.beams.io/posts/git-commit/>`__
+Release
+-------
+
+For versioning we use `setuptools-scm <https://pypi.org/project/setuptools-scm/>`_. This means the version number is
+derived from git tags. To release a new version of the relay on PyPI or Docker Hub, simply tag a commit with a valid version
+number either via git, or from `github <https://github.com/trustlines-protocol/relay/releases/new>`_.
```
Comment (berndbohmeier) [inline README.rst:108]: Link to contracts

File: README.rst [hunk 4]
```diff
@@ -1,4 +1,11 @@
-|Code style: black|
+.. image:: https://img.shields.io/badge/code%20style-black-000000.svg
+    :target: https://github.com/psf/black
+
+.. image:: https://circleci.com/gh/trustlines-protocol/relay.svg?style=svg
+    :target: https://circleci.com/gh/trustlines-protocol/relay
```
Comment (berndbohmeier) [inline README.rst:5]: contracts?

File: CONTRIBUTING.md (hunk 2 above)
Comment (cducrest) [reply CONTRIBUTING.md:45]: You want to say something like: `Implement the corresponding changes in the relay, or open an issue for it`?

Comment (berndbohmeier) [reply CONTRIBUTING.md:45]: yes, or remove it all together. I think it is weird only to mention the clientlib

Comment (cducrest) [reply CONTRIBUTING.md:45]: I removed it.

**Pull request merged**